relay-cardinality/src/redis/quota.rs

```diff
@@ -8,31 +10,34 @@ use crate::window::Slot;
 use crate::{CardinalityLimit, CardinalityScope, OrganizationId, Scoping, SlidingWindow};
 
 /// A quota scoping extracted from a [`CardinalityLimit`] and a [`Scoping`].
-#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
-pub struct QuotaScoping {
+#[derive(Clone, Copy, Debug, Hash, PartialEq, Eq, PartialOrd, Ord)]
+pub struct PartialQuotaScoping {
     pub namespace: Option<MetricNamespace>,
     pub organization_id: Option<OrganizationId>,
     pub project_id: Option<ProjectId>,
     window: SlidingWindow,
+    scope: CardinalityScope,
 }
 
-impl QuotaScoping {
+impl PartialQuotaScoping {
     /// Creates a new [`QuotaScoping`] from a [`Scoping`] and [`CardinalityLimit`].
```

Comment:
```suggestion
    /// Creates a new [`PartialQuotaScoping`] from a [`Scoping`] and [`CardinalityLimit`].
```

Reply:
I really hate that rust analyzers rename ignores doc comments ... ty